src/tests/ftest/launch.py

```diff
@@ -2578,6 +2099,14 @@ def process(self, test, repeat, stop_daos, archive, rename, jenkinslog, core_fil
             return_code |= self._stop_daos_server_service(test)
             return_code |= self._reset_server_storage(test)
 
+        # Mark the test execution as failed if a results.xml file is not found
+        test_logs_dir = os.path.realpath(os.path.join(self.avocado.get_logs_dir(), "latest"))
+        results_xml = os.path.join(test_logs_dir, "results.xml")
+        if not os.path.exists(results_xml):
+            message = f"Missing a '{results_xml}' file for {str(test)}"
```

Comment:
f-string auto converts to string (I think). E.g.
```
>>> v = 1
>>> type(v)
<class 'int'>
>>> f'v = {v}'
'v = 1'
>>> d = {1:1}
>>> f'd = {d}'
'd = {1: 1}'
```
```suggestion
            message = f"Missing a '{results_xml}' file for {test}"
```